burn-import/src/pytorch/recorder.rs

```diff
@@ -115,6 +124,17 @@ impl LoadArgs {
         self.key_remap.push((regex, replacement.into()));
         self
     }
+
+    /// Set top level key to load state_dict from the file.
+    /// Sometimes the state_dict is nested under a top level key in a dict.
+    ///
+    /// # Arguments
+    ///
+    /// * `key` - The top level key to load state_dict from the file.
+    pub fn with_top_level_key(mut self, key: &str) -> Self {
```

Comment:
Reading this I was wondering if something like `with_nested_dict(key)` or `from_nested_dict(key)` would be more clear? The name is open to discussion, just giving my 2 cents :)

Reply:
`with_` prefix is a builder naming convention and I have used top_level_key through already. Not my favorite but consistent.

Reply:
I don't have any better suggestions anyway, so I'll have to agree with the choice 😄